nevergrad/optimization/test_optimizerlib.py

```diff
@@ -114,17 +114,14 @@ def check_optimizer(
     "ASCMA2PDEthird",
     "MultiScaleCMA",
     "PCEDA",
```

Comment:
PCEDA still failed. The others ones may fail as well pretty often if they early converge, they may require adding again to the list of slow optimizers which convergence cannot be tested